packages/pluggableWidgets/video-player-web/src/VideoPlayer.xml

```diff
@@ -8,15 +8,13 @@
     <properties>
         <propertyGroup caption="General">
             <propertyGroup caption="Data source">
-                <property key="urlExpression" type="expression">
```

Comment:
We need to introduce an enumeration:
Name: Type
Values: Dynamic, Expression

If there is values in the expression fields it should be selected expression. If is a new widget, it should be dynamic by default.